cloud/amazon/iam_cert.py

```diff
@@ -228,6 +249,7 @@ def main():
         name=dict(default=None, required=False),
         cert=dict(default=None, required=False),
         key=dict(default=None, required=False),
+        key_contents=dict(default=None, required=False),
         cert_chain=dict(default=None, required=False),
         new_name=dict(default=None, required=False),
         path=dict(default='/', required=False),
```

Comment:
no worries, let us know when it's ready


[image: photo] <http://linkedin.com/in/davilaio>
<http://twitter.com/defionscode>  <http://github.com/defionscode> Jonathan
I. Davila
Principal Architect, Automation Practice, Red Hat, Inc
202.415.1878 // redhat.com // jdavila@redhat.com // Washington, D.C

On Fri, Mar 11, 2016 at 5:03 PM, Scott Brown <notifications@github.com>
wrote:

> In cloud/amazon/iam_cert.py
> <https://github.com/ansible/ansible-modules-core/pull/2976#discussion_r55895517>
> :
>
> > @@ -228,6 +249,7 @@ def main():
> >          name=dict(default=None, required=False),
> >          cert=dict(default=None, required=False),
> >          key=dict(default=None, required=False),
> > +        key_contents=dict(default=None, required=False),
> >          cert_chain=dict(default=None, required=False),
> >          new_name=dict(default=None, required=False),
> >          path=dict(default='/', required=False),
>
> I didn't know mutually_exclusive existed! Thanks for the insight. I'll
> get this patched up and ready to go.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ansible/ansible-modules-core/pull/2976/files#r55895517>
> .
>